### ISSUE TITLE
[BUG] - Incompatible with Packer 1.5.1

### ISSUE THREAD

**Issue by jdost** [vultr/packer-builder-vultr#22]
After my OS upgraded packer to the most recent build (`1.5.1`) using the plugin fails a pretty opaque way.  It seems they made breaking changes in the upgrade from `1.4.5` -> `1.5.0` around interface signatures for builders, which is likely the cause (but the error does not really surface this).

**To Reproduce**
1. Download recent packer version (e.g. `wget https://releases.hashicorp.com/packer/1.5.1/packer_1.5.1_linux_amd64.zip && unzip packer_*_linux_amd64.zip`)
2. Have this plugin compiled and located in the path (e.g. `$HOME/.packer.d/plugins/`)
3. Run a build `packer build -only=vultr path/to/my/packer/config.json`

**Expected behavior**
To successfully run the build (as works with older versions of packer, I downgraded to a cached package and it works again)

**Observed Behavior**
```
$ packer build -only=vultr path/to/config.json
Failed to initialize build 'vultr': error initializing builder 'vultr': fork/exec ~/tmp: permission denied

==> Builds finished but no artifacts were created.
$ PACKER_LOG=1 packer build -only=vultr path/to/config.json
2020/01/09 15:00:26 [INFO] Packer version: 1.5.1 [go1.13.5 linux amd64]
2020/01/09 15:00:26 [DEBUG] Discovered plugin: vultr = ~/.packer.d/plugins/packer-builder-vultr
2020/01/09 15:00:26 using external builders [vultr]
2020/01/09 15:00:26 Attempting to open config file: ~/.packerconfig
2020/01/09 15:00:26 [WARN] Config file doesn't exist: ~/.packerconfig
2020/01/09 15:00:26 Setting cache directory: ~/tmp/packer_cache
2020/01/09 15:00:26 Plugin could not be found at  (exec: "": executable file not found in $PATH). Checking same directory as executable.
2020/01/09 15:00:26 Current exe path: ~/tmp/packer
2020/01/09 15:00:26 Creating plugin client for path: ~/tmp
2020/01/09 15:00:26 Starting plugin: ~/tmp []string{"~/tmp"}
2020/01/09 15:00:26 Build debug mode: false
2020/01/09 15:00:26 Force build: false
2020/01/09 15:00:26 On error: 
2020/01/09 15:00:26 Waiting on builds to complete...
==> Builds finished but no artifacts were created.
2020/01/09 15:00:26 [INFO] (telemetry) Finalizing.
Failed to initialize build 'vultr': error initializing builder 'vultr': fork/exec ~/tmp: permission denied

==> Builds finished but no artifacts were created.
2020/01/09 15:00:26 waiting for all plugin processes to complete...
```
[edited for privacy]

**Comment by ddymko** [comment]
@jdost 

Thanks for spotting this - I'll take a look at addressing this today

**Comment by ddymko** [comment]
@jdost 

I have the changes for the v.1.5.x HCL/Interface updates however there is an bug with packer with 3rd party libs not working properly. It hasn't been merged into a main release yet so I'll work on getting a build with the nightly working.

https://github.com/hashicorp/packer/pull/8543
